src/texprintfsymbols

```diff
@@ -1,13 +1,16 @@
+TEXPRINTF_SYMBOLS
 TEXPRINTF_LW
 TEXPRINTF_FONT
 TEXPRINTF_FCW
 TEXPRINTF_WCW
 TEXPRINTF_ERR
 texprintf
 stexprintf
+texstring
```

Comment:
Whoops, sorry about that.

Reply:
No worries, I didn't notice in the other PR. Until testing the export of `TEXPRINTF_SYMBOLS` I got the error. :b